sdk/formrecognizer/Azure.AI.FormRecognizer/src/ReceiptClient.cs

```diff
@@ -59,6 +59,7 @@ public ReceiptClient(Uri endpoint, FormRecognizerApiKeyCredential credential, Fo
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>A Operation&lt;IReadOnlyList&lt;ExtractedLayoutPage&gt;&gt; to wait on this long-running operation.  Its Operation&lt;IReadOnlyList&lt;ExtractedLayoutPage&gt;&gt;.Value upon successful
         /// completion will contain the extracted receipt.</returns>
+        [ForwardsClientCalls]
```

Comment:
What does this attribute do?

Reply:
It skips diagnostics validation for this method. It's related to the issue we discussed earlier. Basically, the scope names expected by the Test Framework do not correspond to the ones we have, making the tests fail. It seems this is related to the code generator: [#575](https://github.com/Azure/autorest.csharp/issues/575).

Pavel mentioned we should open an issue on our side as well. We'll need to coordinate with CodeGen before GA to get this fixed and remove these attributes.